api.planx.uk/hasura/metadata/index.ts

```diff
@@ -57,7 +57,7 @@ const createScheduledEvent = async (args: RequiredScheduledEventArgs) => {
           value_from_env: "HASURA_PLANX_API_KEY"
         }],
         retry_conf: {
-          num_retries: 3,
+          num_retries: 1,
```

Comment:
this is actually the change that I intended to make earlier in #1273 - which actually only adjusted the Slack _notification_ retry, not the actual send event - it made my testing feedback loop faster here